integrations/influxdb_connector/influxdb-timestream-connector/template.yml

```diff
@@ -0,0 +1,591 @@
+                #foreach($param in $input.params().header.keySet())
+                "$param": "$util.escapeJavaScript($input.params().header.get($param))" #if($foreach.hasNext),#end
+                #end
+              },
+              "pathParameters": {
+                #foreach($param in $input.params().path.keySet())
+                "$param": "$util.escapeJavaScript($input.params().path.get($param))" #if($foreach.hasNext),#end
+                #end
+              },
+              "resource": "$context.resourcePath",
+              "httpMethod": "$context.httpMethod",
+              "body": "$util.escapeJavaScript($input.body)",
+              "queryStringParameters": {
+                #foreach($param in $input.params().querystring.keySet())
+                "$param": "$util.escapeJavaScript($input.params().querystring.get($param))" #if($foreach.hasNext),#end
+                #end
+              },
+              "requestContext": {
+                "accountId": "$context.identity.accountId",
+                "resourceId": "$context.resourceId",
+                "stage": "$context.stage",
+                "requestId": "$context.requestId",
+                "requestTimeEpoch": "$context.requestTimeEpoch",
+                "identity": {
+                  "cognitoIdentityPoolId": "$context.identity.cognitoIdentityPoolId",
+                  "accountId": "$context.identity.accountId",
+                  "cognitoIdentityId": "$context.identity.cognitoIdentityId",
+                  "caller": "$context.identity.caller",
+                  "apiKey": "$context.apiKey",
+                  "sourceIp": "$context.identity.sourceIp",
+                  "cognitoAuthenticationType": "$context.identity.cognitoAuthenticationType",
+                  "cognitoAuthenticationProvider": "$context.identity.cognitoAuthenticationProvider",
+                  "userArn": "$context.identity.userArn",
+                  "userAgent": "$context.identity.userAgent",
+                  "user": "$context.identity.user"
+                },
+                "resourcePath": "$context.resourcePath",
+                "httpMethod": "$context.httpMethod",
+                "apiId": "$context.apiId"
+              },
+              "stageVariables": {
+                #foreach($key in $stageVariables.keySet())
+                "$key": "$util.escapeJavaScript($stageVariables.get($key))" #if($foreach.hasNext),#end
+                #end
+              },
+              "isBase64Encoded": false
+            }
+        Uri: !Sub
+          - arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${LambdaArn}/invocations
+          - LambdaArn: !GetAtt LambdaFunction.Arn
+        RequestParameters:
+          integration.request.header.X-Amz-Invocation-Type: "'Event'"
+        IntegrationResponses:
+          - StatusCode: '202'
+      MethodResponses:
+        - StatusCode: '202' # Asynchronous response, accepted but not yet processed
+
+  AsyncRestApiGatewayDeployment:
```

Comment:
Since a deployment **requires** a method to already exist, and since I have chosen to define two types of method, one for synchronous invocation and one for asynchronous invocation, this leads to some duplication, with matching Async- and Sync- resources.